pkgs/racket-doc/scribblings/reference/sets.scrbl

```diff
@@ -128,6 +128,13 @@ respectively.
 
 }
 
+@defproc[(set->hash [st (or/c set? set-mutable? set-weak?)]) hash?]{
+
+Converts a set to a hash table, if it is @tech{hash set}.
```

Comment:
Wording: "... if it is _a_ hash set"?

Also, it might be worth mentioning that, if the set is mutable, changes to it will be reflected in the returned hash and vice versa.